CI-CD/GitHub/mend-ghaction-conan.yml

```diff
@@ -12,6 +12,7 @@ jobs:
         WS_USERKEY: ${{secrets.PROD_USERKEY}}
         WS_WSS_URL: https://saas.mend.io/agent
 
```

Comment:
For conan scans we should add `WS_RESOLVEALLDEPENDENCIES: false`, as it's a source file scan. It's even more important when conan is installed by Python, otherwise we might also catch the Python packages found in the workspace from the installation (i.e. not delivered with the project)

Reply:
added